libs/SalesforceNetwork/SalesforceNetwork.xcodeproj/xcshareddata/xcschemes/SalesforceNetworkiOS.xcscheme

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "0640"
+   version = "1.3">
+   <BuildAction
+      parallelizeBuildables = "NO"
+      buildImplicitDependencies = "NO">
+      <BuildActionEntries>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "YES"
+            buildForProfiling = "YES"
+            buildForArchiving = "YES"
+            buildForAnalyzing = "YES">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "82D53D301A3618CD00E46F8F"
```

Comment:
This scheme is all wrong anyway - it has references to projects that don't exist, such as ```SalesforceSDKCommon```.